doc/classes/FileAccess.xml

```diff
@@ -99,25 +99,25 @@
 				[b]Note:[/b] Only call [method flush] when you actually need it. Otherwise, it will decrease performance due to constant disk writes.
 			</description>
 		</method>
-		<method name="get_8" qualifiers="const">
+		<method name="get_8" qualifiers="const" deprecated="Use [method get_u8] or [method get_s8] instead.">
 			<return type="int" />
 			<description>
 				Returns the next 8 bits from the file as an integer. See [method store_8] for details on what values can be stored and retrieved this way.
 			</description>
 		</method>
-		<method name="get_16" qualifiers="const">
+		<method name="get_16" qualifiers="const" deprecated="Use [method get_u16] or [method get_s16] instead.">
```

Comment:
Should the deprecated notice point to `get_s16` instead? This was the closest equivalent to it before. (Same for `get_32` and `get_64` below.)

Reply:
The binds are for the `u` versions though

Reply:
I think it makes sense to indicate both options and the user will decide which to use. Maybe the [conversion](https://github.com/godotengine/godot/blob/78ef3b363b8d6fda344716734bef5d746357e291/doc/classes/FileAccess.xml#L462-L494) was used before, which is no longer needed.

Reply:
u version is the equivalent, but I think liks to the both are useful.